### ISSUE TITLE
RPC external feature marker sizing

### ISSUE THREAD

**Issue by tpaukku** [oskariorg/oskari-docs#227]
When adding external svg-icons for vector point features with OskariRPC, it seems that sizePx-attribute (nor size-attribute) has no effect (=external icon is always the size that it physically is and can't be resized).

I have added feature to map with code underneath, but it makes no difference if I change sizePx to 200 or whatever.
Icon itself is displayed nicely, it's only the size that's causing problems.
I have also tried with .png icon and tried using size-attribute (instead of sizePx), but none of these seems to work.
Size-attribute works with Oskari internal svg-icons (shape: 0 for example).
```
this.channel.postRequest("MapModulePlugin.AddFeaturesToMapRequest", [
featureCollectionWrapper,
    {
        layerId: layerTest,
        featureStyle: {
            image: {
            shape: "https://mobile.virma.fi/symbols/Oskari-map-pin.svg",
            sizePx: 20, // Exact icon px size. Used if 'size' not defined.
            }
        }
    }
]);
```

**Comment by ZakarFin** [comment]
Thanks for reporting, someone should take a closer look at this. WFS-styling is kind of a hot topic in Oskari now but I don't know if we have the resources to do anything about this in the near future so no promises :)

**Comment by ZakarFin** [comment]
The `sizePx` is now passed on to `ol/style/Icon` as `size: [sizePx, sizePx]` so it _might_ work now but it hasn't really been tested against this case in particular. The change has been merged to the develop branch (https://github.com/oskariorg/oskari-frontend/blob/f7faab09945ae62c697412022d9a2408155e4928/bundles/mapping/mapmodule/oskariStyle/generator.ol.js#L484) so it will be included in the next release (2.7). Would be great if you get a chance to test this it and confirm if it fixes the issue.

**Comment by ZakarFin** [comment]
Closing as this was fixed
